scripts/pre-push

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node

```

Comment:
Can you please run eslint so that it conforms to the style in the rest of the codebase?